link.py

```diff
@@ -0,0 +1,115 @@
+from collections import deque
+'''
+   Class for a link.
```

Comment:
Class for a link should be on the same line as the triple quotes and use " not '